pkg/runtime/runtime.go

```diff
@@ -30,6 +30,7 @@ import (
 	"time"
 
 	"github.com/cenkalti/backoff/v4"
+	"github.com/dapr/kit/logger"
```

Comment:
this import has to be moved next to the other github.com/dapr imports or the linter will fail.

Reply:
fixed